stories/imdb/stories/filterList.js

```diff
@@ -109,108 +110,124 @@ let schemas = fromPromise(
   )
 )
 
-export default () => (
-  <DarkBox>
-    <Awaiter promise={schemas}>
-      {schemas => (
-        <Provider tree={tree}>
-          <SpacedList>
-            <Query path={['searchRoot', 'searchQuery']} />
-            <Flex>
-              <div style={{ flex: 1 }}>
-                <FilterList
-                  path={['searchRoot', 'criteria']}
-                  fields={schemas.movies.fields}
-                  typeComponents={TypeMap}
-                />
-                <Adder
-                  path={['searchRoot', 'criteria']}
-                  fields={schemas.movies.fields}
-                  uniqueFields
-                />
-              </div>
-              <div style={{ flex: 4, maxWidth: '80%' }}>
-                <ResultCount path={['searchRoot', 'results']} />
-                <DateHistogram
-                  path={['searchRoot', 'releases']}
-                  format={formatYear}
-                />
-                <TermsStatsTable
-                  criteria={['searchRoot', 'criteria']}
-                  path={['searchRoot', 'genreScores']}
-                  tableAttrs={{ style: { margin: 'auto' } }}
-                >
-                  <Column field="key" label="Genre" />
-                  <Column field="count" label="Found" />
-                  <Column
-                    field="key"
-                    label=""
-                    expand={{
-                      display: x =>
-                        `Show top 50 based on meta score for ${x} ▼`,
-                    }}
-                    collapse={{
-                      display: x =>
-                        `Hide top 50 based on meta score for ${x} ▲`,
-                    }}
+const Story = inject(() => {
```

Comment:
instead of indenting this in an inject, move this + observer to the export

Reply:
this actually breaks the story :0